Sources/FluentProvider/Provider.swift

```diff
@@ -175,6 +181,12 @@ public final class Provider: Vapor.Provider {
         if let idKey = self.idKey {
             database.idKey = idKey
         }
+
+        if self.log == true {
+            database.log = { query in
+                drop.log.info(query.description)
```

Comment:
This will be a circular ref if drop holds database, maybe we should throw drop in here weakly just to be safe